extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -106,20 +106,18 @@ function () {
 				self::logUserModelPerformanceData( 'query', 'all_admins' );
 
 				$db = wfGetDB( DB_SLAVE );
-
 				$adminIds = $this->getAdmins();
 				$botIds = $this->getBotIds();
-				$dateTwoYearsAgo = date( 'Y-m-d', strtotime( '-2 years' ) );
+
+				$validAdminIds = array_diff($adminIds,$botIds);
 
 				$sqlData = ( new WikiaSQL() )
 					->SELECT( 'rev_user_text, rev_user, wup_value' )
 					->FROM ( 'revision FORCE INDEX (user_timestamp)' )
```

Comment:
Should we force `user_timestamp` index if we removed timestamp from `WHERE` statement?